python/lsst/obs/base/fitsRawFormatterBase.py

```diff
@@ -271,46 +271,34 @@ def makeFilter(self):
         """
         return lsst.afw.image.Filter(self.observationInfo.physical_filter)
 
-    def readImageComponent(self, component):
-        """Read the image, mask, or variance component of an Exposure.
+    def readComponent(self, component, parameters=None):
```

Comment:
I don't see parameters used anywhere in the method.

Reply:
Correct. The base class requires the argument though since the butler may pass parameters in.  Parameters are not used by any of the raw formatter code so this is a long standing issue. I made a comment on the Jira ticket.  At some point we need to decide what parameters make sense for raw formatter (maybe only bounding box) and complain if any others turn up. Currently the storage class defines the allowed parameters and that might lead to a special RawExposure storage class that defines a different set.